modules/core/src/main/scala/lucuma/odb/api/repo/AsterismRepo.scala

```diff
@@ -34,7 +34,9 @@ sealed trait AsterismRepo[F[_]] extends TopLevelRepo[F, Asterism.Id, AsterismMod
 
   def selectForObservation(oid: Observation.Id, includeDeleted: Boolean = false): F[Option[AsterismModel]]
 
-  def insert(input: AsterismModel.Create): F[AsterismModel]
+  def insert(input: AsterismModel.Create): F[ValidatedInput[AsterismModel]]
+
+  def unsafeInsert(input: AsterismModel.Create): F[AsterismModel]
```

Comment:
The `unsafe` alternatives are helpful for setting up a program programmatically (as is the case in the `Init` code).  In this case, we just want the whole thing to fail if we've made a mistake in wiring it up.